### DISCUSSION TITLE
Suggestion: avoiding the need for two USB cables on LyraT

### DISCUSSION THREAD

**Discussion by technoblogy** [pschatzmann/arduino-audio-tools#717] [Q&A] [answered]
On your page:

https://www.pschatzmann.ch/home/2022/01/24/using-the-esp32-lyrat-audio-board-with-arduino/

you say that you need to connect the LyraT board with two USB cables. In fact there’s a link you can make on the board with a link or solder bridge, close to the POWER and UART USB sockets, that then makes it unnecessary to connect the POWER cable.

**Comment by pschatzmann** [comment, marked as answer]
Thanks for you input. However I prefer to to modify my hardware.
This might be interesting for others however
